src/SFA.DAS.PSRService.Web/Startup.cs

```diff
@@ -98,14 +105,103 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
             if (env.IsDevelopment())
             {
-                app.UseBrowserLink();
+                //app.UseBrowserLink();
```

Comment:
If this is no good then remove it rather than comment it out - or is that a mistake?

Reply:
👍